integrationTests/state/genesisState/genesisState_test.go

```diff
@@ -38,8 +39,11 @@ func TestCreationOfTheGenesisState(t *testing.T) {
 	accountsParser, err := parsing.NewAccountsParser(
 		genesisFile,
 		big.NewInt(6000000000),
+		"erd1lllllllllllllllllllllllllllllllllllllllllllllllllllsckry7t",
```

Comment:
make sure to change to a new address when the genesis minting address is decided.